ansible/roles/data-products-deploy/templates/model-config.j2

```diff
@@ -307,6 +307,12 @@ config() {
 	   	"dialcode-usage-updater")
 		echo '{"search":{"type":"azure","queries":[{"bucket":"'$bucket'","prefix":"derived/dialus/","endDate":"'$endDate'","delta":0}]},"model":"org.ekstep.analytics.updater.UpdateDialcodeUsageDB","modelParams":{},"output":[{"to":"console","params":{"printEvent":false}},{"to":"kafka","params":{"brokerList":"'$brokerList'","topic":"'$learning_topic'"}}],"parallelization":8,"appName":"DialcodeUsageUpdater","deviceMapping":false}'
 		;;
+		"telemetry-replay")
+        echo '{"search":{"type":"azure","queries":[{"bucket":"'$bucket'","prefix":"unique/","endDate":"'$endDate'","delta":0}]},"model":"org.ekstep.analytics.job.TelemetryReplayJob","modelParams":{},"output":[{"to":"console","params":{"printEvent":false}},{"to":"kafka","params":{"brokerList":"'$brokerList'","topic":"'$sinc_topic'"}}],"parallelization":8,"appName":"TelemetryReplayJob","deviceMapping":false}'
```

Comment:
The class which does the replay is called TelemetryReplayJob for both telemetry events and summary events. The name needs to be generic and appropriate.